grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ExplanationTest.java

```diff
@@ -0,0 +1,121 @@
+package ai.grakn.test.graql.reasoner;
```

Comment:
Yeah, the checkers don't read the tests at all. I'm not totally sure it's possible to make them do that...